### DISCUSSION TITLE
Official Connected Apps guide

### DISCUSSION THREAD

**Discussion by hufman** [BimmerGestalt/AAIdrive#255] [Ideas]
BMW/Mini Connected on Android included official support for 3rd-party apps to integrate with the car interface. Sadly this has been deprecated, but here are some notes for reference.

Only the Connected app included the CarAPI SDK to support these external apps. Connected Classic only supported its own internal apps, and MyBMW does not provide the SDK. Since Connected 6.5 no longer starts up the car connection, the best success can be found with Connected 6.4.

- [BMW Connected US 6.4.1](https://apkpure.com/bmw-connected/de.bmw.connected.na/download/6349-APK?from=versions%2Fversion)
- [BMW Connected Europe 6.4.1](https://apkpure.com/bmw-connected/de.bmw.connected/download/6349-APK?from=versions%2Fversion)
- [Mini Connected US 6.4.1](https://apkpure.com/mini-connected/de.mini.connected.na/download/6349-APK?from=versions%2Fversion)
- [Mini Connected Europe 6.4.1](https://apkpure.com/mini-connected/de.mini.connected/download/6349-APK?from=versions%2Fversion)

There were only a limited selection of compatible Android apps, much fewer than the compatible iPhone apps. One additional major difference from iPhone is that BMW will only start up the Android music apps if the car is running the Apps connection over Bluetooth.

- [Spotify 8.5.66](https://spotify.en.uptodown.com/android/download/2292403)
- [iHeartRadio 9.27.0](https://apkpure.com/iheartradio-radio-podcasts-music-on-demand/com.clearchannel.iheartradio.controller/download/709270003-APK?from=versions%2Fversion)

Pandora and Samsung SmartThings were also supported, but I haven't identified the specific last version.

**Comment by jagerx9** [comment]
Just want to say thank you for your efforts. I was wondering why the app suddenly stopped working for days at a time. Otherwise it works great for me. Going to downgrade to Connected 6.4
